chia/_tests/cmds/wallet/test_tx_decorators.py

```diff
@@ -20,7 +20,7 @@ def test_cmd(**kwargs: Any) -> list[TransactionRecord]:
 
     runner: CliRunner = CliRunner()
     with runner.isolated_filesystem():
-        runner.invoke(test_cmd, ["--transaction-file", "./temp.transaction"])
+        runner.invoke(test_cmd, ["--transaction-file-out", "./temp.transaction"])
```

Comment:
Note that I am aware this is a command line API change, but a sort of necessary one from an initial bad decision that I think will cause minimal headache.